test/check-machines-disks

```diff
@@ -837,12 +839,13 @@ class TestMachinesDisks(machineslib.VirtualMachinesCase):
             mode='use-existing',
         ).execute()
 
-        # Detach the iscsi disk before reaching teardown because shutting of the domains would sometimes hang when iscsi disks are attached
+        # Detach the iscsi disk before reaching teardown because shutting of the domains would sometimes hang when
+        # iscsi disks are attached.
         m.execute("virsh detach-disk subVmTest1 --target vdb")
 
         # AppArmor doesn't like the non-standard path for our storage pools
         if m.image in ["debian-testing"]:
-            self.allow_journal_messages(f'.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="{self.vm_tmpdir}.*')
+            self.allow_journal_messages(f'.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="{self.vm_tmpdir}.*')  # noqa: E501
```

Comment:
FTR: Agreed to explicitly breaking the max line length for these.

Reply:
We can also try to remove them, how do you feel about that? Would be a separate PR and maybe future work? (when this occurs again)

Reply:
Yes, the test could just put them into the correct dir -- /var/lib/libvirt/images/ or so?

Reply:
I.e. fine for a follow-up, doesn't need to happen here.